.github/workflows/database-backup.yml

```diff
@@ -4,72 +4,68 @@ concurrency: build_and_deploy_main
 on:
   workflow_dispatch:
     inputs:
+      environment:
+        description: Environment to backup
+        required: true
+        default: staging
+        type: choice
+        options:
+        - 'staging'
+        - 'production'
       overwriteThisMorningsBackup:
         required: true
         type: boolean
         default: false
+      PTRdatabase:
+        description: Are you backing up a PTR database copy
+        required: false
+        type: boolean
+        default: false
   schedule: # 03:00 UTC
     - cron: '0 3 * * *'
 
+env:
+  SERVICE_NAME: itt-mentor-services
+  SERVICE_SHORT: ittms
+  TF_VARS_PATH: terraform/application/config
+
 jobs:
   backup:
-    name: Backup AKS Database (production)
+    name: Backup AKS Database
     if: ${{ github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && github.event.inputs.overwriteThisMorningsBackup == 'true') }}
     runs-on: ubuntu-latest
     environment:
-        name: production
+      name: ${{ inputs.environment || 'production' }}
+    env:
+      DEPLOY_ENV: ${{ inputs.environment || 'production'  }}
 
     steps:
     - uses: actions/checkout@v4
       name: Checkout
 
-    - uses: azure/login@v2
-      with:
-        creds: ${{ secrets.AZURE_CREDENTIALS }}
-
-    - name: Setup postgres client
-      uses: DFE-Digital/github-actions/install-postgres-client@master
-      with:
-        version: 14
-
-    - name: Install kubectl
-      uses: azure/setup-kubectl@v4
-      with:
-        version: "v1.26.1" # default is latest stable
-
-    - uses: DFE-Digital/github-actions/set-kubelogin-environment@master
-      with:
-        azure-credentials: ${{ secrets.AZURE_CREDENTIALS }}
-
-    - name: K8 setup
-      shell: bash
-      run: |
-        make ci production get-cluster-credentials
-        make bin/konduit.sh
-
-    - name: Set environment variable
-      run: echo "BACKUP_FILE_NAME=ittm_prod_$(date +"%F")" >> $GITHUB_ENV
-
-    - name: Backup Prod DB
+    - name: Set environment variables
       run: |
-        bin/konduit.sh -t 7200 itt-mentor-services-production -- pg_dump -E utf8 --clean --if-exists --no-owner --verbose --no-password -f ${BACKUP_FILE_NAME}.sql
-        tar -cvzf ${BACKUP_FILE_NAME}.tar.gz ${BACKUP_FILE_NAME}.sql
+        source global_config/${{ env.DEPLOY_ENV }}.sh
+        tf_vars_file=${{ env.TF_VARS_PATH }}/${{ env.DEPLOY_ENV }}.tfvars.json
+        echo "CLUSTER=$(jq -r '.cluster' ${tf_vars_file})" >> $GITHUB_ENV
+        echo "RESOURCE_GROUP_NAME=${AZURE_RESOURCE_PREFIX}-${SERVICE_SHORT}-${CONFIG_SHORT}-rg" >> $GITHUB_ENV
+        echo "STORAGE_ACCOUNT_NAME=${AZURE_RESOURCE_PREFIX}${SERVICE_SHORT}dbbkp${CONFIG_SHORT}sa" >> $GITHUB_ENV
+        TODAY=$(date +"%F")
+        echo "BACKUP_FILE=${SERVICE_SHORT}_${CONFIG_SHORT}_${TODAY}.sql" >> $GITHUB_ENV
+        echo "DB=${AZURE_RESOURCE_PREFIX}-${SERVICE_SHORT}-${CONFIG_SHORT}-pg" >> $GITHUB_ENV
```

Comment:
Could we rename to DB_SERVER for clarity and consistency?

Reply:
done